sections/image-with-text.liquid

```diff
@@ -433,6 +433,12 @@
           "type": "url",
           "id": "button_link",
           "label": "t:sections.image-with-text.blocks.button.settings.button_link.label"
+        },
+        {
+          "type": "checkbox",
+          "id": "button_style_secondary",
```

Comment:
Interesting, so the logic to apply the class was in there but not the setting ? 😅